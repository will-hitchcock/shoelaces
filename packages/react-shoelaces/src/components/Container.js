import styled from 'styled-components'
import { container } from 'shoelaces'

export const Container = styled.div`${container}`

export default Container
