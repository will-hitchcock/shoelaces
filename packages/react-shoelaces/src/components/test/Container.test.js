import Container from '../Container'

describe('<Container />', () => {
  it('should render a <Container />', () => {
    expect(shallow(<Container />)).toMatchSnapshot()
  })
})
