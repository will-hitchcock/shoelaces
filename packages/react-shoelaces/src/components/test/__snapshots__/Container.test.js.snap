// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render a <Container /> 1`] = `
<div
  className="sc-bdVaJa kDHesU"
/>
`;
